### PR TITLE
CI: upgrade common code updates

### PR DESCRIPTION
**Pull request by Jaakko-Os**
-upgrade_common refactoring
-authentication and TLS disabled

### CODE REVIEW

**Review by Xenwar — APPROVED** [review]

File: scripts/feature_tests/setup_env.sh [hunk 1]
```diff
@@ -9,5 +9,9 @@ if [[ "${1}" == "ug" ]]; then
   # shellcheck source="$M3PATH/scripts/feature_tests/upgrade/upgrade_vars.sh"
   source "${M3PATH}/scripts/feature_tests/upgrade/upgrade_vars.sh"
 fi
+if [[ "${1}" == "no-tls" ]]; then
+  export IRONIC_BASIC_AUTH="false"
+  export IRONIC_TLS_SETUP="false"
```
Comment (smoshiur1237) [inline scripts/feature_tests/setup_env.sh:14]: Please add nodeDrain variable.
export NODE_DRAIN_TIMEOUT=300s

**Review by smoshiur1237 — APPROVED** [review]

File: Makefile [hunk 2]
```diff
@@ -32,6 +32,9 @@ setup_env:
 setup_env_ug:
 	./scripts/feature_tests/setup_env.sh "ug"
 
+setup_env_no_tls:
+	./scripts/feature_tests/setup_env.sh "no-tls"
```
Comment (Xenwar) [inline Makefile:36]: we do not need this file.
we are ok with TLS enabled for now.

File: scripts/feature_tests/setup_env.sh [hunk 3]
```diff
@@ -9,5 +9,10 @@ if [[ "${1}" == "ug" ]]; then
   # shellcheck source="$M3PATH/scripts/feature_tests/upgrade/upgrade_vars.sh"
   source "${M3PATH}/scripts/feature_tests/upgrade/upgrade_vars.sh"
 fi
+if [[ "${1}" == "no-tls" ]]; then
```
Comment (Xenwar) [inline scripts/feature_tests/setup_env.sh:12]: we do not need this file.
we are ok with TLS enabled for now.

**Review by Xenwar — CHANGES_REQUESTED** [review]

File: Makefile (hunk 2 above)
Comment (Jaakko-Os) [reply Makefile:36]: it can be cleaned up later.

File: scripts/feature_tests/setup_env.sh (hunk 3 above)
Comment (Jaakko-Os) [reply scripts/feature_tests/setup_env.sh:12]: to be cleaned up later.

Comment (smoshiur1237) [reply scripts/feature_tests/setup_env.sh:12]: It will not make any change to the current features.

File: Makefile (hunk 2 above)
Comment (smoshiur1237) [reply Makefile:36]: It will not make any change to the current features.

Comment (smoshiur1237) [reply Makefile:36]: So we can keep that.

File: scripts/feature_tests/setup_env.sh (hunk 1 above)
Comment (Jaakko-Os) [reply scripts/feature_tests/setup_env.sh:14]: done

Comment (Jaakko-Os) [reply scripts/feature_tests/setup_env.sh:14]: done

**Pull request merged**